### ISSUE TITLE
Deserialization error: Wrong directory fields

### ISSUE THREAD

**Issue by loheander** [m3adow/docker-letsencrypt-simp_le#3]
Hi!

Even though it seems that this image has switched to zenhacks fork, I still get the "Deserialization error: Wrong directory fields" error when running the image. Maybe it is caused by the most recent automated builds failing? https://hub.docker.com/r/m3adow/letsencrypt-simp_le/builds/

**Comment by lyoshenka** [comment]
Yea, looks like the new build isn't working. From the Docker build log:

```
ERROR: unsatisfiable constraints:

  py-pip (virtual):
    provided by: py2-pip
    required by: world[py-pip]

Removing intermediate container 02fad8c5b532

The command '/bin/sh -c apk --update add python 		python-dev 		py-setuptools 		py-pip 		openssl-dev 		openssl 		musl-dev 		gcc 		libffi-dev 		darkhttpd 	&& wget -qO- https://codeload.github.com/zenhack/simp_le/tar.gz/master | tar xz 	&& pip install -e /simp_le-master/ 	&& mkdir /certs 	&& apk --purge del musl-dev openssl-dev libffi-dev gcc python-dev py-pip' returned a non-zero code: 1
```

**Comment by m3adow** [comment]
I've created a testing Docker image for that (m3adow/letsencrypt-simp_le:testing). Please test if you're able to run everything as you're used to and everything's fixed. I haven't been using this image for a long time and therefore can't test it myself.

**Comment by setiseta** [comment]
the testing tag is working thx.

is there a reason you not use this one for yourself? are there other / better / easier ways?

**Comment by m3adow** [comment]
Thanks for your feedback @setiseta.

Why do I not use it? The server I initially created this image for up for was a personal test server which I stopped using over a year ago. Since then I've transitioned into playing with CoreOS and K8s and I haven't yet reached the state of a production ready system where I'm actually deploying applications.

**Comment by m3adow** [comment]
Closed with 49fc84d.
